src/clr-core/package.js

```diff
@@ -0,0 +1,32 @@
+'use strict';
+
+const fs = require('fs-extra');
+const path = require('path');
+const del = require('del');
+
+// Temporary script needed to remove cache files and
+// files Angular CLI leaves in referenced dependencies
+// https://github.com/ng-packagr/ng-packagr/pull/1372
+// https://github.com/ng-packagr/ng-packagr/issues/1318
+// https://github.com/angular/angular/issues/33395
+// https://justinfagnani.com/2019/11/01/how-to-publish-web-components-to-npm/
+const read = dir =>
+  fs
+    .readdirSync(dir)
+    .reduce(
+      (files, file) =>
+        fs.statSync(path.join(dir, file)).isDirectory()
+          ? files.concat(read(path.join(dir, file)))
+          : files.concat(path.join(dir, file)),
+      []
+    );
+
+del.sync(['./dist/clr-core/**/*.{tsbuildinfo,ngsummary.json}', './dist/clr-core/*.{tsbuildinfo,ngsummary.json}']);
+
+read('./dist/clr-core')
+  .filter(f => f.includes('package.json'))
+  .forEach(file => {
+    const data = fs.readJsonSync(file);
+    ['__processed_by_ivy_ngcc__', 'scripts'].forEach(p => delete data[p]);
+    fs.writeJsonSync(file, data, { spaces: 2 });
+  });
```

Comment:
Angular CLI and TS enjoy adding random cache files and mutating our package json files during the build. This script cleans it all up after for deployment.